src/components/LandingPage/HeroSection.tsx

```diff
@@ -0,0 +1,198 @@
+
+  useEffect(() => {
+    const target = new Date(eventDateFormatted);
+
+    const interval = setInterval(() => {
+      const now = new Date();
+      const difference = target.getTime() - now.getTime();
+
+      const d = Math.floor(difference / (1000 * 60 * 60 * 24));
+      setDays(d);
+
+      const h = Math.floor(
+        (difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60),
+      );
+      setHours(h);
+
+      const m = Math.floor((difference % (1000 * 60 * 60)) / (1000 * 60));
+      setMinutes(m);
+
+      const s = Math.floor((difference % (1000 * 60)) / 1000);
+      setSeconds(s);
+      if (d <= 0 && h <= 0 && m <= 0 && s <= 0) {
+        setHackathonTime(true);
+      }
+    }, 1000);
+    return () => clearInterval(interval);
+  }, []);
+
+  return (
+    <div className={HERO_TILE_STYLES}>
+      <div>
+        <h1
+          className="mt-16 flex-wrap text-5xl font-black text-[#FFFF] drop-shadow-lg md:text-center md:text-6xl"
+          style={{
+            textShadow: `
+    -2px -2px 0 #7055FD, 
+    2px -2px 0 #7055FD, 
+    -2px 2px 0 #7055FD, 
+    2px 2px 0 #7055FD,
+    -2px -2px 0 #7055FD,
+    2px -2px 0 #7055FD,
+    -2px 2px 0 #7055FD,
+    2px 2px 0 #7055FD
+`,
+          }}
+        >
+          {" "}
+          {eventName}
+          <span className="text-[#BAFBE4]"> {eventYear}</span>
+        </h1>
+        <strong className="text-1xl my-4 flex flex-wrap justify-center text-[#7055FD] opacity-95  md:text-center md:text-xl lg:px-40">
+          {eventBlurb}
+        </strong>
+      </div>
+      <div className={LINK_STYLES}>
+        <Link href="/" legacyBehavior>
+          <div className=" mb-4 rounded-2xl border-4 border-white bg-[#7055FD] px-6 py-2 text-sm text-white  hover:opacity-70 md:mb-0 md:px-6">
+            Join Hackathon
+          </div>
+        </Link>
+      </div>
+      <div className={LINK_STYLES}>
+        <p className="my-2">
+          Already registered?
+          <Link href="/" legacyBehavior>
+            <span className=" text-[#7055FD]  hover:opacity-70"> Sign in</span>
+          </Link>
+        </p>
+      </div>
+
+      <div className={WEBPAGE_CONTAINER}>
+        <div className="relative rounded-t-md border-t-[30px] border-white">
+          <Image
+            src={WINDOW_BUTTONS_SVG}
+            alt="check mark icon"
+            width={50}
+            height={50}
+            className="absolute left-0 top-0 -mt-5 ms-3"
+          ></Image>
+          <div className=" mx-auto p-4">
+            <div className="m-2 h-[24rem] rounded-3xl bg-[#BAFBE4] px-10 opacity-90 md:m-5">
+              <h1 className="pt-10 text-center text-2xl font-bold text-[#7055FD]">
+                {eventName} begins...{" "}
+              </h1>
+              {hackathonTime ? (
+                <h1>
+                  {eventName} {eventYear} has begun!
+                </h1>
+              ) : (
+                <div className={COUNTDOWN_CONTAINER}>
```

Comment:
the countdown is pretty big so I think we should separate it into another component like `CountdownTimer`